clusterctl.yaml.tmpl

```diff
@@ -0,0 +1,23 @@
+CLUSTERCTL_LOG_LEVEL: 10
+
+NUTANIX_ENDPOINT: ""    # IP or FQDN of Prism Central
+NUTANIX_USER: ""        # Prism Central user
+NUTANIX_PASSWORD: ""    # Prism Central password
+NUTANIX_INSECURE: false # Set to true for when using a self-signed certificate for Prism Central
+NUTANIX_PRISM_ELEMENT_CLUSTER_NAME: ""
+NUTANIX_SUBNET_NAME: ""
+
+KUBERNETES_VERSION: "v1.27.8"           # Kubernetes version, must match the version in NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME
+NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME: "" # OS image name, must match the version in KUBERNETES_VERSION
+CONTROL_PLANE_ENDPOINT_IP: ""           # An available IP that will be used by kube-vip for the Kubernetes API Server
+NUTANIX_SSH_AUTHORIZED_KEY: ""          # Public key to use with user 'capiuser' on all Machines
+WORKER_MACHINE_COUNT: 2                 # Number of Machines in the worker node pool
+
+KUBEVIP_LB_ENABLE: "false"  # See https://kube-vip.io/docs/about/architecture/?query=lb_enable#control-plane-load-balancing
+KUBEVIP_SVC_ENABLE: "false" # See https://kube-vip.io/docs/about/architecture/?query=lb_enable#kubernetes-service-load-balancing
+
+providers:
+  # add a custom provider
+  - name: "nutanix"
+    url: "file://${HOME}/.cluster-api/overrides/infrastructure-nutanix/v0.0.0/infrastructure-components.yaml"
```

Comment:
clusterctl was failing to use `$HOME` env because of the extra `/`.

From https://cluster-api.sigs.k8s.io/clusterctl/configuration
>Note: It is possible to use the ${HOME} and ${CLUSTERCTL_REPOSITORY_PATH} environment variables in url.